proto/Makefile

```diff
@@ -16,17 +16,15 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PWD=$(shell pwd)
-
-PROTOC      := $(PWD)/../../../../../bin/protoc
+PROTOC      := $(GOPATH)/bin/protoc
```

Comment:
I've sort-of punted on this by returning the old logic that uses relative paths for GOPATH, which is what godeps.sh does. Hopefully we can use GOBIN to fix this stuff globally in the future.